client/src/pages/pitchDoc/PitchDoc.tsx

```diff
@@ -153,37 +178,39 @@ function PitchDoc(): ReactElement {
   /**
    * Calls /pitch api through api wrapper to get ALL pitches
    */
-  const getAllUnclaimedPitches = async (): Promise<void> => {
-    const resp = await getApprovedPitches();
-
-    if (!isError(resp) && resp.data) {
-      setApprovedPitches(resp.data.result);
-    }
-  };
-
-  useEffect(() => {
-    getAllUnclaimedPitches();
-  }, []);
-
-  /**
-   * Searches through all of the pitches to find pitches by name
-   */
   const handleSearch = useCallback(
     (searchTerm: string): IPitch[] => {
-      let searchPitches = [...approvedPitches];
+      let searchPitches = [...displayedCards];
       searchTerm = searchTerm.toLowerCase().trim();
 
-      searchPitches = approvedPitches.filter((pitch: IPitch) => {
+      searchPitches = displayedCards.filter((pitch: IPitch) => {
         // Avoid null fields in database
         const pitchName = pitch.name ? pitch.name.toLowerCase() : '';
         return pitchName.includes(searchTerm);
       });
 
       return searchPitches;
     },
-    [approvedPitches],
+    [displayedCards],
   );
 
+  const getAllUnclaimedPitches = useCallback(async (): Promise<void> => {
+    let resp;
+    if (isAdmin) {
+      resp = await getPendingPitches();
+    } else {
+      resp = await getApprovedPitches();
+    }
+
+    if (!isError(resp) && resp.data) {
+      setDisplayedCards(resp.data.result);
+    }
+  }, [isAdmin]);
+
+  useEffect(() => {
```

Comment:
Why is this function called in a separate `useEffect` hook? Since its only dependency is the function, I feel like it makes more sense to call this function in the on page load `useEffect` on line 118.